gobcore/model/sa/gob.py

```diff
@@ -29,6 +34,25 @@ def get_column(column_name, column_specification):
     return column
 
 
+def _fields_for_collection(collection):
```

Comment:
What is the difference between this one model['all_fields']
model['all_fields'] is defined in GOBModel.__init__

Reply:
Good catch. Does the exact same thing. Removed _fields_for_collection.